app/assets/javascripts/ama_layout/mailcheck/autocorrect_email.coffee

```diff
@@ -0,0 +1,81 @@
+class AMALayout.EmailSuggestion
```

Comment:
Nope, not sure why it looks like it... from history:
![image](https://cloud.githubusercontent.com/assets/473578/14299855/08fd7746-fb4a-11e5-99ea-5471a0411dd5.png)